crates/wasi-http/src/http_impl.rs

```diff
@@ -157,16 +195,17 @@ impl<T: WasiHttpView> WasiHttpViewExt for T {
                 let (s, conn) = t?;
                 tokio::task::spawn(async move {
                     if let Err(err) = conn.await {
-                        println!("Connection failed: {:?}", err);
+                        println!("[host/client] Connection failed: {:?}", err);
```

Comment:
can you please swap all `println!` in this file to `tracing::debug!`? I hadn't realized those were still in here.